services/storage-calculator/calculate-storage.sh

```diff
@@ -59,6 +59,15 @@ echo "$ALL_ENVIRONMENTS" | jq -c '.data.environments[] | select((.environments |
 
     echo " > $CONSOLE_URL - $PROJECT_NAME: environment $ENVIRONMENT_NAME"
 
+    KUBECTL="kubectl --insecure-skip-tls-verify --token=$CONSOLE_TOKEN --server=$CONSOLE_URL -n $ENVIRONMENT_NAMESPACE"
+
+    # add label to namespaces to indicate the storagecalculator status
+    if [[ $STORAGE_CALC != "1" ]] ; then
+      ${KUBECTL} label namespace $ENVIRONMENT_NAMESPACE lagoon.sh/storageCalculatorDisabled="true" --overwrite
+    else
+      ${KUBECTL} label namespace $ENVIRONMENT_NAMESPACE lagoon.sh/storageCalculatorDisabled="false" --overwrite
+    fi
+
     if [[ $STORAGE_CALC != "1" ]] ; then
       echo " > $CONSOLE_URL - $PROJECT_NAME - $ENVIRONMENT_NAME: storage calculation disabled, skipping"
```

Comment:
This isn't quite true anymore (the skipping), right? We're more marking the ns as having `lagoon.sh/storageCalculatorDisabled" as false.